<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QWidget" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="tableauFacture">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>tableau facture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="objectFacture">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>objectFacture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FactureManage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>facture Management</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
